pages/[env]/index.js

```diff
@@ -534,16 +539,16 @@ export default function MainPage() {
                       </HStack>
                       {Object.entries(
                         state.inFlightRequests?.[cleanAddress(account)] || {}
-                      ).map(([signatureRequestId, compositeKeys]) => (
-                        <>
+                      ).map(([signatureRequestId, compositeKeys], i) => (
+                        <Stack key={i}>
```

Comment:
if `signatureRequestId` is unique then it's better to use it as a `key` instead of array index `i`, as there could be an incorrect render flow with array indices
```suggestion
                      ).map(([signatureRequestId, compositeKeys], i) => (
                        <Stack key={signatureRequestId}>
```